### DISCUSSION TITLE
How to customize the default wiki plugins list?

### DISCUSSION THREAD

**Discussion by talha131** [tiddly-gittly/TidGi-Desktop#275] [Q&A] [answered]
When a new Wiki is created, it is prepopulated with existing plugins and themes.

How do I customize the default template Wiki so that all new wikis will have only the plugins I have chosen?

**Comment by linonetwo** [comment, marked as answer]
Well...This is a feature that I haven't have time to make in https://github.com/tiddly-gittly/Awesome-TiddlyWiki-Editions

I'd like tidgi to read this edition list, or allow you create your own local edition for new wiki.
